stdlib/universe/join_use_previous_test.flux

```diff
@@ -13,34 +13,34 @@ inData =
 #default,,,,,,
 ,result,table,_time,value,_field,_measurement
 ,_result,0,2020-05-14T17:23:00Z,1,a,a
-,_result,0,2020-05-14T17:40:00Z,2,a,a
-,_result,0,2020-05-14T17:41:00Z,3,a,a
-,_result,0,2020-05-14T17:42:00Z,4,a,a
-,_result,0,2020-05-14T17:43:00Z,5,a,a
-,_result,0,2020-05-14T17:44:00Z,6,a,a
-,_result,0,2020-05-14T17:45:00Z,,a,a
-,_result,0,2020-05-14T17:46:00Z,7,a,a
-,_result,0,2020-05-14T17:47:00Z,,a,a
-,_result,0,2020-05-14T17:48:00Z,8,a,a
-,_result,0,2020-05-14T17:49:00Z,9,a,a
-,_result,0,2020-05-14T17:50:00Z,10,a,a
+,_result,1,2020-05-14T17:40:00Z,2,a,a
```

Comment:
Could you please elaborate on why you are saving each row in a separate table?
Here is the [issue](https://github.com/influxdata/flux/issues/2864) for which this test was initially created.

Reply:
Honestly I don't know exactly what is going here since it seems to have been changed twice since it was introduced. I tried to tweak it to be closer to how it was originally, but I still had to keep some changes to ensure `range` does not error due to duplicate group keys :|

Reply:
https://github.com/influxdata/flux/pull/5162/commits/6e8d0a90e89710809a60ab6e11dc8953f7a4369d